eng/pipelines/templates/stages/archetype-autorest-preview.yml

```diff
@@ -204,7 +204,7 @@ stages:
 
     # Create authenticated .npmrc file for publishing
     - ${{ if eq(parameters.PublishInternal, 'true') }}:
-      - template: ../steps/create-authenticated-npmrc.yml
+      - template: ../../../common/pipelines/templates/steps/create-authenticated-npmrc.yml
```

Comment:
```suggestion
      - template: /eng/common/pipelines/templates/steps/create-authenticated-npmrc.yml
```

Reply:
This template will be used by external repos.  We can't use an `@self` prefix and I think we should stay relative.

Reply:
Won't every place that uses this have /eng/common? I feel like relative paths are much more fragile in devops template usage but both should work so not a big deal.

Reply:
absolute paths worked across repos, even for repos without a local eng/common